config/nodeKeys.js

```diff
@@ -4,11 +4,16 @@ if (process.env.NODE_ENV === "production") {
     NODE_PACKAGE_ID: "e9c55b2c-4019-463e-8efa-622f23221402",
     NODE_LISTING_RESOURCE: "9be4623a-3c01-4b2d-9c7b-567a41abbc1c",
     NODE_PHONE_RESOURCE: "2f66ad5d-5066-49d8-846a-6751cfd23863",
+    /* Heroku creates an environment variable called DATABASE_URL for postgres which is subject to change at any time by Heroku */
+    /* This means we need to use the DATABASE_URL environment variable rather than hard-coding the connection string */
+    PG_CONNECTION_STRING: process.env.DATABASE_URL
```

Comment:
Would be good to include where to find this in the wiki/readme.

Reply:
Added